moveit_core/trajectory_processing/src/iterative_time_parameterization.cpp

```diff
@@ -35,6 +35,7 @@
 /* Author: Ken Anderson */
```

Comment:
Yeah, I figured this to be more of an initial testing version, but if it can actually be merged I will move it to another file and give the plugin another name.